kahuna/public/js/services/preset-label.js

```diff
@@ -0,0 +1,39 @@
+import angular from 'angular';
+
+var presetLabelService = angular.module('kahuna.services.presetLabel', []);
+
+presetLabelService.factory('presetLabelService',
+                            ['$window', function ($window) {
+
+    const presetLabelsKey = 'preset labels';
+
+    function getLabels() {
+        return JSON.parse($window.localStorage.getItem(presetLabelsKey));
+    }
+
+    function addLabels(newLabels) {
+        let labels = new Set(getLabels());
```

Comment:
This can be `const`. When you use the `add` / `remove` it's not actually mutating the object (which is what makes these quite nice).